src/ripple/basics/impl/contract.cpp

```diff
@@ -39,6 +41,20 @@ accessViolation() noexcept
 
 } // detail
 
+void
+LogThrow (char const* title)
+{
```

Comment:
If this takes a `std::string` instead we can build up a much richer message at the throw site.